tools/stability-checker/local/input/testing.sh

```diff
@@ -185,6 +185,10 @@ echo "- Testing Logging components..."
 helm test logging
 loggingTestErr=$?
 
+echo "- Testing Monitoring components..."
+helm test monitoring
+loggingTestErr=$?
```

Comment:
Suggested change `loggingTestErr=$?` to `monitoringTestErr=$?`

Reply:
Also add `| [ ${monitoringTestErr} -ne 0 ]` to line, https://github.com/kyma-project/kyma/blob/b6c67df052edcfba3e203a5e5592e6f076a03f58/tools/stability-checker/local/input/testing.sh#L216

Reply:
yeah thats right :) I wanted to add the same for logging too if it works for prometheus

Reply:
fixed.